Berksfile

```diff
@@ -3,6 +3,7 @@ source 'https://supermarket.getchef.com'
 metadata
 
 cookbook 'yum', '~> 3.0'
+cookbook 'exhibitor', git: 'git@github.com:SimpleFinance/chef-exhibitor.git'
```

Comment:
Hey thanks for the review!  For some reason the exhibitor cookbook is not in the Chef supermarket so the best we can do right now is explicitly reference the master branch in github and then pin a version in the metadata at 0.4.0 which I have done with my latest commit to avoid unintended version drift in the future.  There were some important fixes in 0.4.0 that were needed so that's why I settled on that version.  If the exhibitor cookbook ever makes it up to the supermarket one day the dependency should be a lot simpler to define.  Thanks again for the extra set of eyes.